test/e2e/multiple_zk_test.go

```diff
@@ -80,7 +82,7 @@ func testMultiZKCluster(t *testing.T) {
 	defaultCluster.Status.Init()
 	defaultCluster.ObjectMeta.Name = "zk3"
 	defaultCluster.Spec.Persistence.VolumeReclaimPolicy = "Delete"
-
+	defaultCluster.Spec.Replicas = 1
```

Comment:
Why reduce replica count from 3 to 1 ?

Reply:
I had reduced it as the test was taking more than 10 minutes to run and it was getting timed out I found out a way to solve it and have changed it back to 3 again.